styles/index.css

```diff
@@ -13,3 +13,9 @@
   width: 40rem;
   height: 40rem;
 }
+
+.card-container {
+  max-width: 400rem;
```

Comment:
Are you sure about setting max-width to 400rem?

Reply:
Thanks for pointing it out, it was a mistake, i was trying to check something, changed back to 50 rem now